src/open_inwoner/kvk/views.py

```diff
@@ -33,9 +35,14 @@ def get_form_kwargs(self):
 
         kvk_client = KvKClient()
 
-        company_branches = kvk_client.get_all_company_branches(
-            kvk=self.request.user.kvk
-        )
+        try:
+            company_branches = kvk_client.get_all_company_branches(
+                kvk=self.request.user.kvk
+            )
+        except KVKAPIException as exc:
+            messages.error(request=self.request, message=exc.default_message)
```

Comment:
The exception message is internal:

```suggestion
            messages.error(request=self.request, message="We are temporarily unable to show your branches, please try again at a later point.")
```

Reply:
(I also think we deprecated `default_message` in an earlier iteration).